editor/code/package.json

```diff
@@ -173,5 +190,11 @@
     "typecheck": "tsc -b",
     "compile": "npm run esbuild",
     "watch": "npm run esbuild -- --watch"
+  },
+  "__metadata": {
```

Comment:
What is this?

Reply:
VSCode generated it by accident, I'll remove it